test/features/deploy_travel_agency.feature

```diff
@@ -52,7 +54,32 @@ Feature: Deploy Travel agency service and verify its functionality
 
     Then Service "travels" with process name "travels" is available
 
-  Scenario: Travel application with required Visa
+	Examples:
+      | native   | minutes | build-request-cpu | 
+      | disabled | 10      | 1                 | 
+
+	# Disabled as long as https://issues.redhat.com/browse/KOGITO-1893  is not merged or https://issues.redhat.com/browse/KOGITO-1917 is not fixed
+	@disabled
+    @native
+    Examples:
+      | native   | minutes | build-request-cpu | 
+      | enabled  | 30      | 4                 | 
```

Comment:
With CPU 4, the build takes more than 20 min. That's why I set 30 min here.
Without CPU, the build takes more than 30 min).